docs/development_suite/api-console/api-design/services.md

```diff
@@ -179,54 +179,108 @@ For each container port, you have to define:
+
+#### From ConfigMap
+
+Use this type to load a file of a ConfigMap into an environment variable; the **Key**, **ConfigMap Name** and **ConfigMap File Name** must be provided, along with an optional **Description**.
+
+The ConfigMap can be chosen among the ones configured in the Project configuration.
 
 :::warning
+
 We always suggest mounting ConfigMaps on the files system and using environment variables *From ConfigMap* when necessary and only with small-sized files.
+
 :::
 
 :::info
 Please refer to the dedicated [ConfigMaps](/development_suite/api-console/api-design/services.md#configmaps) and [Secrets](/development_suite/api-console/api-design/services.md#secrets) sections for more information about how to manage them using the Console.
 :::
 
-Therefore, for each variable, you have to define:
+#### From Downward API
 
-* **Key** (*required*)
+This type represents a value that is obtained from the Kubernetes [Downward API](https://kubernetes.io/docs/concepts/workloads/pods/downward-api/).
 
-* **Value Type** (*required*)
+Downward API allows you to expose information about a Pod to itself, such as the Pod's name, namespace, IP address, resource limits, label and annotations values or other information.
 
-* **Value** (*not required, present only if **Value Type** is Plain Text*)
+##### Why should I use the Downward API?
 
-* **Secret Name** (*present and required only if **Value Type** is From Secret*)
+The fact that the information are provided through environment variables **decouples the application in the container from the Kubernetes API**, 
+because environment variables are a standard way to provide configuration to applications.
 
-* **Secret Key** (*present and required only if **Value Type** is From Secret*)
+This is particularly useful in scenarios where your application needs to know information about the Pod it is running in, 
+but you don't want it or you can't to access the Kubernetes APIs to obtain them or to hardcode that information into the application itself.
 
-* **ConfigMap Name** (*present and required only if **Value Type** is From ConfigMap*)
+It is also useful for accessing to Console Project specific information, which are always available
+through dedicated [annotations](/development_suite/api-console/api-design/services.md#annotations-configuration) and [labels](/development_suite/api-console/api-design/services.md#labels-configuration).
 
-* **ConfigMap File Name** (*present and required only if **Value Type** is From ConfigMap*)
+This can allow you, for example, to know the name of the Console Project where the Microservice is deployed, the name of the environment and other information 
+that you can use for monitoring, logging or even application logic purposes.
 
-* **Description**
+Pods for example can also communicate their name to other Pods in the same namespace, so that they can be used as a unique identifier for purposes such as leader-election among multiple replicas.
```

Comment:
```suggestion
Pods for example can also communicate their name to other Pods in the same namespace, so that it can be used as a unique identifier for purposes such as leader-election among multiple replicas.
```